client/src/Components/SliderPanel.jsx

```diff
@@ -0,0 +1,145 @@
+import {React, useState} from "react";
+import Slider from '@mui/material/Slider';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import MuiInput from '@mui/material/Input';
+import { styled } from '@mui/material/styles';
+
+function valuetext(value) {
+    return `${value}`;
+}
+
+export default function SliderPanel(props) {
+
+    const Input2 = styled(MuiInput)`width: 110px;`;
+    const [newValue, setInputVal] = useState(props.min);
+
+    const mark1 = [{
+        value: 50000,
+        label: '50000'},
+    {
+        value: 240000,
+        label: '240000'},
+    {
+        value: 430000,
+        label: '430000'},
+    {
+        value: 620000,
+        label: '620000'},
+    {
+        value: 810000,
+        label: '810000'},
+    {
+        value: 1000000,
+        label: '1000000'}
+    ];
+    
+    const mark2 = [{
+        value: 0,
+        label: '0'},
+    {
+        value: 6,
+        label: '6'},
+    {
+        value: 12,
+        label: '12'},
+    {
+        value: 18,
+        label: '18'},
+    {
+        value: 24,
+        label: '24'},
+    {
+        value: 30,
+        label: '30'}
+    ];
+
+
+    const handleSliderChange = (event, newValue) => {
+        props.setVal(newValue);
+        setInputVal(props.min);
+    };
```

Comment:
this function should be at index.jsx and used here as props.handleSliderChange()